pkg/util/webconsole/proxy_address.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2024 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package webconsole
```

Comment:
Currently this package is only used for the web-console feature, but its functions primarily deal with obtaining proxy addresses. So to better reflect its purpose, I think `proxyaddr` makes more sense as its package name, with relocation to the `util/kube/` dir.

Reply:
Changed to `pkg/util/kube/proxyaddr`.